### ISSUE TITLE
amb_eye_lo

### ISSUE THREAD

**Issue by kike-0304** [ZiqiaoPeng/SyncTalk#59]
# aud att loss (regions out of face should be static)
        if self.opt.amb_aud_loss and not self.opt.torso:
            ambient_aud = outputs['ambient_aud']
            loss_amb_aud = (ambient_aud * (~lowface_mask.view(-1))).mean()
            # gradually increase it
            lambda_amb = step_factor * self.opt.lambda_amb 
            loss += lambda_amb * loss_amb_aud

        # eye att loss
        if self.opt.amb_eye_loss and not self.opt.torso:
            ambient_eye = outputs['ambient_eye']
            loss_cross = ((ambient_eye)*(lowface_mask.view(-1))).mean()
            lambda_amb = step_factor * self.opt.lambda_amb
            loss += lambda_amb * loss_cross
